mmv1/templates/terraform/pre_create/network_services_gateway.go.tmpl

```diff
@@ -0,0 +1,3 @@
+if obj["type"] == "SECURE_WEB_GATEWAY" && obj["routingMode"] == nil {
+    return fmt.Errorf("Field routing_mode is required for Gateway of type SECURE_WEB_GATEWAY")

```

Comment:
1. Does the server side have the similar validation when creating a new resource? If yes, the validation is not needed in the Terraform provider.

2. Does the server side have the similar validation when updating other fields in the resource?

Reply:
Not needed